src/mpid/ch4/src/ch4r_request.h

```diff
@@ -57,6 +57,67 @@ static inline MPIR_Request *MPIDI_CH4I_am_request_create(MPIR_Request_kind_t kin
     goto fn_exit;
 }
 
+#undef FUNCNAME
+#define FUNCNAME MPIDI_CH4I_am_request_init
+#undef FCNAME
+#define FCNAME MPL_QUOTE(FUNCNAME)
+MPL_STATIC_INLINE_PREFIX MPIR_Request *MPIDI_CH4I_am_request_init(MPIR_Request * req,
+                                                                  MPIR_Request_kind_t kind,
+                                                                  int ref_increment)
+{
+    int i;
+
+    MPIR_FUNC_VERBOSE_STATE_DECL(MPID_STATE_MPIDI_CH4I_AM_REQUEST_INIT);
+    MPIR_FUNC_VERBOSE_ENTER(MPID_STATE_MPIDI_CH4I_AM_REQUEST_INIT);
+
+    MPIR_Assert(req != NULL);
+    for (i = 0; i < ref_increment - 1; i++)
```

Comment:
Oh, actually my intention was to change interpretation of `ref_increment` too (if `ref_increment == 1`, do add_ref once). Anyway, I'll approve this for now and later submit a PR to fix this.